packages/lesswrong/server/permissions/accessFilters.ts

```diff
@@ -10,6 +10,11 @@ import { constantTimeCompare } from "@/lib/helpers";
 import { userCanDo, userIsAdmin, userIsAdminOrMod, userOwns } from "@/lib/vulcan-users/permissions";
 import _ from "underscore";
 
+const automatedContentEvaluationCheckAccess: CheckAccessFunction<'AutomatedContentEvaluations'> = async (currentUser, document, context): Promise<boolean> => {
+  if (!currentUser || !document) return false;
+  return currentUser.isAdmin
```

Comment:
This should be `userIsAdmin(currentUser)`;

Reply:
I did that first, but everywhere else in this file we use the pattern of currentUser.isAdmin